client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
```

Comment:
I've made these changes in this PR, just so that we don't have to revisit the files on here.